app/util/date.ts

```diff
@@ -47,4 +43,22 @@ export const timeAgoAbbr = (d: Date, options?: FormatDistanceToNowStrictOptions)
     },
   })
 
-export const formatDateTime = (d: Date) => format(d, 'MMM d, yyyy H:mm aa')
+const dateOptions = {
+  month: 'short',
+  day: 'numeric',
+  year: 'numeric',
+} satisfies Intl.DateTimeFormatOptions
+
+const timeOptions = {
+  hour: 'numeric',
+  minute: 'numeric',
+} satisfies Intl.DateTimeFormatOptions
```

Comment:
Heh, check this out: I was just reading the MDN page and noticed the style shortcuts section: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#datestyle

I think we can reduce these to `timeStyle: 'short'` and `dateStyle: 'medium'`! That is clean.

```ts
const dateOptions = { dateStyle: 'medium' } satisfies Intl.DateTimeFormatOptions
const timeOptions = { timeStyle: 'short' } satisfies Intl.DateTimeFormatOptions
```

<img width="637" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/c780ca4e-4cd3-40e3-8c8d-b3363ce4d5e7">

Oddly it does change one of the results, the French one. Seems fine to me!

<img width="880" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/886f83b6-0d5c-4dcb-a17f-5f776dfa1a48">

Reply:
Oh, interesting. Funny that it changes the German syntax. I don't have strong feelings on it, but agree the shorter configuration syntax is nice. Will get that in.

Reply:
Eh, I maybe prefer the output of the earlier version, but am fine either way, and recognize this will likely be an edge case for some time. We can always change it back if we decide that's better.
<img width="532" alt="Screenshot 2024-04-16 at 9 32 25 PM" src="https://github.com/oxidecomputer/console/assets/22547/8dee2d78-03df-49be-afb0-75bb7451cb82">

Reply:
Even though it looks worse (maybe) I like the idea of taking the details out of our hands — in theory they should know better than we do what’s idiomatic in each locale.

Reply:
Makes sense; I agree with that.